assets/src/application/ui/display/icons/svgs/types.ts

```diff
@@ -4,3 +4,6 @@ export interface IconProps {
 }
 
 export type IconSize = 'big' | 'bigger' | 'small' | 'smaller' | 'tiny';
+
+export type HorizontalMargin = 'extreme' | 'huge' | 'big' | 'bigger' | 'small' | 'smaller' | 'tiny' | 'micro' | 'nano';
```

Comment:
later this can be moved into a more centralized place or derived from smth like `MarginSizes`

Reply:
Not sure if an absolute value may be better here

Reply:
yes, this is a good idea, the reason I've went with these values is that these are matching with the CSS variables, which have absolute values set in `:root`;
need to hear what preference @tn3rb also have

Reply:
yes please use variables for things like this because then the values can be changed via a theme.
Eventually the goal is to create the ability for users to create their own themes, which will really only be control over a dozen or so parameters, but enough to tweak EE's display to better match their site/preferences.